react/features/local-recording/recording/flac/flacEncodeWorker.js

```diff
@@ -0,0 +1,416 @@
+import {
+    MAIN_THREAD_FINISH,
+    MAIN_THREAD_INIT,
+    MAIN_THREAD_NEW_DATA_ARRIVED,
+    WORKER_BLOB_READY,
+    WORKER_LIBFLAC_READY
+} from './messageTypes';
+
+/**
+ * WebWorker that does FLAC encoding using libflac.js
+ */
+
+/* eslint-disable */
+importScripts('/libs/libflac3-1.3.2.min.js');
+/* eslint-enable */
+
+// There is a number of API calls to libflac.js, which does not conform
+// to the camalCase naming convention, but we cannot change it.
+// So we disable the ESLint rule `new-cap` in this file.
+/* eslint-disable new-cap */
+
+// Flow will complain about the number keys in `FLAC_ERRORS,
+// ESLint will complain about the `declare` statement.
+// As the current workaround, add an exception for eslint.
+/* eslint-disable flowtype/no-types-missing-file-annotation*/
+declare var Flac: Object;
```

Comment:
How much work would it take to fix the flow errors instead of working around them?